PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -8,20 +10,38 @@ public static void Main(string[] args)
         Question[] questions = LoadQuestions(filePath);
 
         int numberCorrect = 0;
+        int numberIncorrect = 0;
         for (int i = 0; i < questions.Length; i++)
         {
             bool result = AskQuestion(questions[i]);
             if (result)
             {
                 numberCorrect++;
             }
+            else
+            {
+                numberIncorrect++;
+            }
         }
+        
+        
+        // added a message to let the user know that the trivia has been completed along with some additional stats
+        Console.WriteLine("\nYou have completed the trivia!");
```

Comment:
nit: `Environment.NewLine` over `\n`
```suggestion
        Console.WriteLine($"{Environment.NewLine}You have completed the trivia!");
```